apps/docs/src/app/platform/component-docs/platform-button/platform-button-docs.component.ts

```diff
@@ -51,13 +58,14 @@ export class PlatformButtonDocsComponent implements OnInit {
     data: any = {
         properties: {
             label: 'click here',
-            type: 'default',
-            option: 'default',
+            buttonType: 'default',
             width: '100px',
             size: 'default',
             icon: '',
-            compact: false,
-            disabled: false
+            contentDensity: 'false',
```

Comment:
Why it is boolean value as it is type string you can initialised to empty string.

Reply:
It is added for examples, the default value will be cozy